objects/screen.c

```diff
@@ -1655,6 +1658,60 @@ luaA_screen_get_name(lua_State *L, screen_t *s)
     return 1;
 }
 
+/** Get the content of the active screen as a cairo surface
+ *
+ * @return A cairo surface
+ * @staticfct content
```

Comment:
Since it's a property ("dynamic property"?), we shouldn't document it as a method that returns something, but as a property of a given type :

```suggestion
 * @property content
 * @tparam surface A cairo surface with the screen content.
 * @see gears.surface
```

(+ I have added the same @see as client.content property)

Reply:
Same comment. However the project likes things documented should be done.